CONTRIBUTING.md

```diff
@@ -99,8 +99,8 @@ Output format of any non-plumbing command can change in non-backward-compatible
    a [git-machete package](https://aur.archlinux.org/packages/git-machete) is hosted in Arch User Repository (AUR).
    If the release introduces significant changes/critical bugfixes, please [flag the package as out of date](https://aur.archlinux.org/pkgbase/git-machete/flag).
 
-1. Verify if changes you made in [blog-1.md](https://github.com/VirtusLab/git-machete/blob/develop/docs/blog-1/blog-1.md)
-   and [blog-2.md](https://github.com/VirtusLab/git-machete/blob/develop/docs/blog-2/blog-2.md)
+<!-- 1. Verify if changes you made in [blog-1.md](https://github.com/VirtusLab/git-machete/blob/develop/blogs/git-machete-1/blog-1.md)
```

Comment:
Actually, same point applies to the Docker blogs now (although keeping them up to date is arguably of lesser importance than the git-machete blogs which kinda serve as the main online documentation, at least until #106 is addressed)

Reply:
I will add it, but with some formatting since now the list of articles and files in fairly long.